Code/Editor/LogFile.cpp

```diff
@@ -178,6 +181,140 @@ void CLogFile::FormatLineV(const char * format, va_list argList)
     CryLog("%s", szBuffer);
 }
 
+namespace LogFileInternal
+{
+    // Stores information about the OS queried from WMI
+    struct OSInfo
+    {
+        // Stores the Name property from Win32_OperatingSystem
+        AZStd::string m_name;
+        // Stores the Version property from Win32_OperatingSystem
+        AZStd::string m_version;
+    };
+
+    OSInfo QueryOSInfo()
+    {
+        // The GetVersion WinAPI function returns the version of Windows
+        // that was manifested at the time the application was built
+        // and not the version of Windows that is currently running
+        // https://learn.microsoft.com/en-us/windows/win32/api/sysinfoapi/nf-sysinfoapi-getversion
```

Comment:
:O lol thanks for fixing this!
One suggestion is that this comment is a bit confusing out of the context of this PR, since GetVersion is no longer being used. How about this?

```suggestion
        // This specifically avoids using the GetVersion/GetVersionEx WinAPI 
        // functions because those will only return the version of Windows
        // that was manifested at the time the application was built
        // and not the version of Windows that is currently running
        // https://learn.microsoft.com/en-us/windows/win32/api/sysinfoapi/nf-sysinfoapi-getversion
```

Reply:
That sounds good.